core/tests/ocr_dump_tests.py

```diff
@@ -50,7 +51,8 @@ def test_new_dump(self):
         t = tarfile.open(dump.path, "r:bz2")
         members = t.getmembers()
         self.assertEqual(len(members), 112) # ocr xml and txt for each page
-        self.assertEqual(members[0].size, 26282)
+        # Now each record is much shorter because the text field is removed
```

Comment:
This comment does not make sense without the context of your tasks. Please remove it.

Reply:
I'll remove it